scripts/bridgeERC20.ts

```diff
@@ -37,7 +37,7 @@ const FUEL_FUNGIBLE_TOKEN_ADDRESS = process.env.FUEL_FUNGIBLE_TOKEN_ADDRESS || '
   const fuelAccount = env.fuel.signers[1];
   const fuelAccountAddr = fuelAccount.address.toHexString();
   const fuelMessagePortal = env.eth.fuelMessagePortal.connect(ethAccount);
-  const gatewayContract = env.eth.l1ERC20Gateway.connect(ethAccount);
+  const gatewayContract = env.eth.fuelERC20Gateway.connect(ethAccount);
```

Comment:
Same nit about the use of "erc20" terminology.

Reply:
It's the contracts name. See previous comments